### ISSUE TITLE
The /blog page on the wp site needs better margins

### ISSUE THREAD

**Issue by GrumpyOldTroll** [cvuuf/church#47]
The blue bars go right up to the edge of the text.
(Jill submitted)

![1556663511463blob](https://user-images.githubusercontent.com/361578/57574515-4d229200-73ef-11e9-8dda-b224ab98ddbf.jpg)

**Comment by shaneniebergall** [comment]
I'm happy to try to fix this page, but honestly I don't think it belongs. There's already a better link to https://www.chaliceuu.org/category/news/ from the menus and sidebar, and there's no need to point to a page that has uncategorized posts on it.. @GrumpyOldTroll let's discuss this next meeting.

**Comment by GrumpyOldTroll** [comment]
That is a fair point, but it made me wonder how she got to this page, which seems like the real problem under that view point.  So I searched for "blog/" and found it used here:
https://www.chaliceuu.org/blog/newsletter/

Changed it to use "/category/news/" instead of "/blog/", and since that seems (I think?) to be the only link to it, I agree it seems ok to ignore the /blog/ page, and that it's not really meant to be a user-facing page when it shows every post regardless of category.

So just linking to the right page instead makes good sense to me:

![image](https://user-images.githubusercontent.com/361578/74792168-78d18300-5271-11ea-8284-cd95b864694d.png)

Any future links to this kind of ugly page that ideally wouldn't be exposed probably should be treated as a bug in the link.

I guess another choice would be to add a redirect to the .htaccess, in case anybody hits the bad link some other way, tho maybe I'll defer that unless somebody else manages to hit that page, unless you think it would be useful.
